lib/DxilPIXPasses/DxilDebugInstrumentation.cpp

```diff
@@ -1515,8 +1511,6 @@ bool DxilDebugInstrumentation::RunOnFunction(Module &M, DxilModule &DM,
     }
   }
 
-  DM.ReEmitDxilResources();
```

Comment:
It is now the caller's (PIX's) responsibility to call the emit-metadata pass after invoking this pass.